docs/versioning.md

```diff
@@ -0,0 +1,42 @@
+# Kubernetes API and Release Versioning
+
+Legend:
+
+* **Kube &lt;major&gt;.&lt;minor&gt;.&lt;patch&gt;** refers to the version of Kubernetes that is released. This versions all components: apiserver, kubelet, kubectl, etc.
+* **API vX[betaX]** refers to the version of the HTTP API.
+
+## Release Timeline
+
+### Minor version timeline
+
+* Kube 1.0
```

Comment:
We've been using three sets of digits for releases thus far. Should this be 1.0.0 (to be pedantic) rather than 1.0?